src/lib/routeNames.js

```diff
@@ -3,7 +3,7 @@ export const ROUTE_NAMES = {
   //
   INVITE_TICKET: Symbol('INVITE_TICKET'),
   INVITE_TRANSACTIONS: Symbol('INVITE_TRANSACTIONS'),
-  INVITES_SEND: Symbol('INVITES_SEND'),
+  INVITE: Symbol('INVITE'),
```

Comment:
Why the renaming? I liked the explicitness, since there's a good few pages relating to invites. (Though this should probably be renamed still per the next commit, to `INVITE_EMAIL` or similar.)

Reply:
In the latest commit I pulled the router out into a separate hook and made it more generic so that it can be used in a nested fashion as well. In the mocks we have a navigation to invite but then within the Invite page a subsection of the page navigates between types of invites. So for the toplevel router, the only route it needs to know about is the "Invite" route and then the Invite view has its own router to navigate between states. Only the primary router captures the browser back button, etc.